website/content/docs/enterprise/license/utilization-reporting.mdx

```diff
@@ -0,0 +1,290 @@
+---
+layout: docs
+page_title: Automated license utilization reporting
+description: >-
+  Learn what data HashiCorp collects to meter Enterprise license utilization. Enable or disable reporting. Review sample payloads and logs.
+---
+
+# Automated License utilization reporting
+
+Automated license utilization reporting sends license utilization data to
+HashiCorp without requiring you to manually collect and report them. It also
+lets you review your license usage with the monitoring solution you already use
+(for example Splunk, Datadog, or others) so you can optimize and manage your
+deployments. Use these reports to understand how much more you can deploy under
+your current contract, protect against overutilization, and budget for predicted
+consumption.  
+
+Automated reporting shares the minimum data required to validate license
+utilization as defined in our contracts. They consist of mostly computed metrics
+and will never contain Personal Identifiable Information (PII) or other
+sensitive information. Automated reporting shares the data with HashiCorp using
+a secure, unidirectional HTTPS API and makes an auditable record in the product
+logs each time it submits a report.
```

Comment:
We might want to mention that the reporting is done on a 24 hour basis. We mention below that the reporting starts within 24 hours, but I didn't notice where we say that the reporting happens once a day, everyday.